src/repeat.ts

```diff
@@ -22,6 +22,8 @@ import {
 import {viewsRequireLifecycle} from './analyze-view-factory';
 import {AbstractRepeater} from './abstract-repeater';
 
+const matcherExtractionMarker = '__marker_extracted__';
```

Comment:
Is there any downside to using a `Symbol` for the `matcherExtractionMarker ` property name? `Symbol` props are not enumerable, but that shouldn't be an issue here, since `Repeat` is not meant to be serialized (?).

Reply:
In this case, we are adding it on an instruction, and that's quite hidden away from apps, I think we are safe. For symbol vs string in general, I think it's about what's (probably?) more performant, easy to use (Aurelia polyfills does come with Symbol, but it's not used anywhere), so one can run an Aurelia app without having to polyfill a lot of stuff. This is true for all parts of our v1 core modules, we are not using Symbols.

Reply:
Excellent point! I didn't realize there was no IE 11 support. Nice catch 💪